doc/man1/openssl-enc.pod

```diff
@@ -418,6 +423,9 @@ certain parameters. So if, for example, you want to use RC2 with a
 
 The default digest was changed from MD5 to SHA256 in OpenSSL 1.1.0.
 
+The openssl enc -ciphers option was deprecated in OpenSSL 1.1.1
+in favor of the openssl enc -list option.
```

Comment:
I'd like to suggest a rephrase.

```
The B<-list> option was added in OpenSSL 3.0.

The B<-ciphers> option was deprecated in OpenSSL 3.0.
```

Reply:
Shouldn't it be 1.1.1 for the `-list` option, not 3.0?

Reply:
Uhmmmmmm...  as far as I can read code, the *current* option in 1.1.1 *is* `-ciphers`, not `-list`

Reply:
Yes.  I still thing it should disappear as and when we can (which is fornever).

Reply:
> Uhmmmmmm... as far as I can read code, the _current_ option in 1.1.1 _is_ `-ciphers`, not `-list`

The plan is to name it `-list` for both for consistency in 1.1.1 (which agrees with the fact that it's called `OPT_LIST` internally anyway) and to leave `-ciphers` as an alias, which is deprecated in 3.0.  I'd like to emphasize: it's the status quo which is inconsistent, and inconsistency of commands and options has been a long standing nuisance of the openssl app IMHO.

Reply:
Please take a look at what I added in my latest update.  I believe we are saying the same thing but just want to confirm:
```
The -ciphers option was deprecated in OpenSSL 3.0 in favor of
the -list option, which was added in OpenSSL 1.1.1.
```

Reply:
Meanwhile, I prefer @levitte's formulation with two separate sentences. Only the first version number needs to be corrected:

```
The B<-list> option was added in OpenSSL 1.1.

The B<-ciphers> option was deprecated in OpenSSL 3.0.
```

The reason is because it was me myself (see #7854 or commit fc5ecaddd0c2), who suggested that more stereotypes should be used in the HISTORY section in order to make it easier for programmers to find the relevant information.  Sorry for the ping pong.

Reply:
If you absolutely want to say that it was added in a previous version, then please say `1.1.1e`!  `1.1.1` is an actual release (the first in the 1.1.1 series), and *`-list` doesn't exist there*

Reply:
Good point.

Reply:
This has been updated in my latest commit.